src/targeting/youtube-ima.ts

```diff
@@ -0,0 +1,60 @@
+import type { CustomParams, MaybeArray } from '../types';
+import { filterValues } from './build-page-targeting';
+
+/**
+ * @param  {Record<string, MaybeArray<string|number|boolean>>
+ * Follows https://support.google.com/admanager/answer/1080597
+ */
+const encodeVastTagKeyValues = (
+	query: Record<string, MaybeArray<string>>,
+): string => {
+	const unencodedUrl = Object.entries(query)
+		.map(([key, value]) => {
+			const queryValue = Array.isArray(value)
+				? value.join(',')
+				: String(value);
+			return `${key}=${queryValue}`;
+		})
+		.join('&');
+	return unencodedUrl
+		.replace(/=/g, '%3D')
+		.replace(/&/g, '%26')
+		.replace(/,/g, '%2C');
+};
+
+const buildImaAdTagUrl = (
+	adUnit: string,
+	customParams: CustomParams,
+): string => {
+	// TODO: Add regular build-page-targeting to
+	// get all targeting including ab participations.
+	// For now hardcode adtest (at)
+	const mergedCustomParams = { ...customParams, at: 'fixed-puppies' };
```

Comment:
Are we not ready to remove the ad test parameter to show real ads?

Reply:
Not yet but close - we have a separate PR in progress to derive `ab-participations` in commercial-core: #699 

I thought we could release this first and enhance as other targeting gets rolled into commercial-core. It should work the same as now - the test has not started yet as we are waiting for a bugfix from Google.